R/states.R

```diff
@@ -0,0 +1,52 @@
+#' @title Define model states
+#' @description Create_states creates and initialises the human states for
+#'  the model
+#'
+#' @title Create and initialise states
+#'
+#' @param psq the model parameters
+#'
+#' @return states
```

Comment:
Should this function be exported? (add `#' export` and redocument)

Reply:
done